sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsResultCollection.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.util.IterableStream;
+
+/**
+ * A collection model that contains a list of {@link T} along with model version and batch's statistics
+ *
+ * @param <T> any child class of {@link TextAnalyticsResult}.
```

Comment:
nit: change to "any type that extends {@link TextAnalyticsResult}."

Reply:
We are going to follow the same pattern as .NET does, so create a concrete collection type for each endpoint but not a generic type, TextAnalyticsResultCollection. The reason behind it is, in the future, it could be difference between vary ResultCollection types.

Reply:
Outdated. This generic class no longer exist.